spec/features/staff/users_can_add_benefitting_countries_spec.rb

```diff
@@ -0,0 +1,106 @@
+RSpec.feature "users can add benefitting countries as intended beneficiaries" do
+  context "when the user is signed as a BEIS user" do
+    let(:user) { create(:beis_user) }
+    before { authenticate!(user: user) }
+    let(:activity) { create(:activity, :at_geography_step, organisation: user.organisation, intended_beneficiaries: nil) }
+
+    context "if they choose 'recipient country' as geography option" do
+      context "after choosing one country" do
+        scenario "the user has the option of selecting other intended beneficiaries based on a reduced list depending on the region of the country selected" do
+          visit activity_step_path(activity, :geography)
+          choose "Country"
+          click_button t("form.button.activity.submit")
+          expect(page).to have_select(t("form.label.activity.recipient_country"))
+
+          select "Bolivia"
+          click_button t("form.button.activity.submit")
+          expect(activity.reload.recipient_region).to eq("489") # South America
+
+          expect(page).to have_content t("form.legend.activity.intended_beneficiaries")
+          expect(page).to have_content("Argentina")
+          expect(page).to have_content("Venezuela")
+          check "Argentina"
+          check "Colombia"
+          check "Peru"
+          click_button t("form.button.activity.submit")
+          activity.reload
+          expect(activity.intended_beneficiaries).to eq(["AR", "CO", "PE"])
```

Comment:
Will the array always be stored in alphabetical order?

Reply:
The countries will be displayed in the UI in alphabetical order, and I _think_ that's the way they'll get stored